### ISSUE TITLE
AVT 1 - Select has DAP violations when disabled <select> is used

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4537]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component and select the disable <select> knob
Run DAP and the following violations are present (see screenshot):
<img width="943" alt="Screen Shot 2019-11-01 at 4 01 24 PM" src="https://user-images.githubusercontent.com/21676914/68056918-62072080-fcc2-11e9-8c95-f0efb6875f37.png">

Note: The select is missing aria-disabled=true" or disabled flag which is causing this issue.

**Comment by snidersd** [comment]
<img width="943" alt="Screen Shot 2019-11-01 at 4 01 24 PM" src="https://user-images.githubusercontent.com/21676914/68056918-62072080-fcc2-11e9-8c95-f0efb6875f37.png">

**Comment by dakahn** [comment]
As discussed offline disabled components are exempt from 1.4.3's contrast requirements. Looks like an error with DAP 👍
